api/migrations/versions/472bf293e0a1_.py

```diff
@@ -0,0 +1,22 @@
+"""empty message
+
+Revision ID: 472bf293e0a1
+Revises: ('79c61673a487', '7e8bffa88454')
+Create Date: 2019-06-18 11:32:36.203013
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '472bf293e0a1'
+down_revision = ('79c61673a487', '7e8bffa88454')
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    pass
```

Comment:
Also this one?

Reply:
Resolving. Thanks @jaderabbit

Reply:
DO NOT RESOLVE NOW! IT'S ALREADY DEPLOYED! Eeek